articles/quickstarts/qrng.md

```diff
@@ -48,16 +63,33 @@ We can use this representation to visualize what the code is doing:
 
 <img src="~/media/qrng-H.png" width="450" alt="Preparing a qubit in superposition">
 
-
 * Then we measure the qubit and save the output:
 
 <img src="~/media/qrng-meas.png" width="450" alt="Measuring a qubit and saving the output">
 
 Since the outcome of the measurement is completely random, we have obtained a random bit. We can call this operation several times to create integers. For example, if we call the operation three times to obtain three random bits, we can build random 3-bit numbers (that is, a random number between 0 and 7).
 
-## Creating a complete random number generator using a host program
 
-Now that we have a Q# operation that generates random bits, we can use it to build a complete quantum random number generator with a host program.
+## Creating a complete random number generator
+
+Now that we have a Q# operation that generates random bits, we can use it to build a complete quantum random number generator. We can use the Q# command line applications or use a host program. 
+
```

Comment:
```suggestion

To create the full Q# command line application, add the following entry point to your Q# program: 
```qsharp
        @EntryPoint()
        operation Main() : Int {
            let max = 10;
            Message($"Random number between 0 and {max}: ");  
        
            let nBits = Floor(Log(IntAsDouble(max))/LogOf2() + 1.);
    
            mutable bits = new Result[0];
            for(bit in 1 .. nBits) {
                set bits += [SampleQuantumRandomNumberGenerator()];
            }
    
            let output = ResultArrayAsInt(bits); 
            return output <= max ? output | Main();
        }
```
```

Reply:
@bettinaheim, that should be in the relevant sample, no? Looks like https://github.com/microsoft/Quantum/blob/71f3068ffd62f12351341ed53f9443db520d13eb/samples/getting-started/qrng/Qrng.qs#L18-L32 is the relevant range, such that ``::code language="qsharp" source="~/quantum/samples/getting-started/qrng/Qrng.qs" range="18-32":::` should do the trick here.